huaweicloud/resource_huaweicloud_bcs_instance.go

```diff
@@ -0,0 +1,760 @@
+package huaweicloud
+
+import (
+	"bytes"
+	"fmt"
+	"log"
+	"strings"
+	"time"
+
+	"github.com/hashicorp/terraform-plugin-sdk/helper/resource"
+	"github.com/hashicorp/terraform-plugin-sdk/helper/schema"
+	"github.com/hashicorp/terraform-plugin-sdk/helper/validation"
+	"github.com/huaweicloud/golangsdk"
+	"github.com/huaweicloud/golangsdk/openstack/bcs/v2/blockchains"
+	"github.com/huaweicloud/golangsdk/openstack/dms/v1/instances"
+	"github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/config"
+	"github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/utils"
+)
+
+func resourceBCSInstanceV2() *schema.Resource {
+	return &schema.Resource{
+		Create: resourceBCSInstanceV2Create,
+		Read:   resourceBCSInstanceV2Read,
+		Update: resourceBCSInstanceV2Update,
```

Comment:
Since there is nothing to do in update func, this line is unnecessary

Reply:
This function is used to update `delete_obs` and `delete_storage`, and the related notes added.